src/main/java/core/mate/academy/service/MachineProducer.java

```diff
@@ -1,11 +1,12 @@
 package core.mate.academy.service;
 
+import core.mate.academy.model.Machine;
 import java.util.List;
 
-public interface MachineProducer {
+public interface MachineProducer<T extends Machine> {
     /**
      * In your implementations - create 2-3 models, add them to the list and return
      * @return - the list of models
      */
-    List<Object> get();
+    List<? extends Machine> get();
```

Comment:
With this implementation, you can return List<Truck> inside ExcavatorProducer 
![image](https://github.com/mate-academy/jv-pecs/assets/20326858/f89912b0-82c1-4d39-82c3-69113d4724ab)